minio/minio
# Synchronize bucket cycle numbers

## Description

Synchronize bucket cycles so it is much more likely that the same prefixes will be picked up for scanning.

Use the global bloom filter cycle for that. Bump bloom filter versions to clear those.

Minor cleanups.

## How to test this PR?

Set `_MINIO_SERVER_DEBUG=on`. Observe that `data-usage: Cycle: 3040, Entries:...` log entries are the same across cluster.

## Types of changes
